otter_welcome_buddy/common/constants.py

```diff
@@ -9,10 +10,20 @@ class CronExpressions(Enum):
 
 COMMAND_PREFIX: str = "!"
 
+DATA_DIR: str = "data"
+DATA_FILE: str = "otter-buddy.db"
+DATA_FILE_PATH: str = os.path.join(DATA_DIR, DATA_FILE)
+
+ALL_DIRS = (
+    attrib_value
+    for attrib_name, attrib_value in list(globals().items())
+    if attrib_name.endswith("DIR")
+)
+
 # Discord role that give access to admin role based commands
-OTTER_ADMIN = "ROOT"
+OTTER_ADMIN: str = "ROOT"
 # Discord role that give access to moderator role based commands
-OTTER_MODERATOR = "Collaborator"
+OTTER_MODERATOR: str = "Collaborator"
 # Discord role that give access to the remaining channels and is
 # given when the user react to WELCOME_MESSAGES
-OTTER_ROLE = "Interviewee"
+OTTER_ROLE: str = "Interviewee"
```

Comment:
Why not interviewee?

Reply:
Same than other comment, is more generic (and because I'll change Interviewee for Member in some moment soon on the server.